### PR TITLE
fix: fix placeholder color search input

### PR DESCRIPTION
**Pull request by caesar-exploit**
Rich, I am trying to connect to the "repro" I have good ideas to improve but I still don't understand why I can't sync my commits. I hope to solve this soon or if someone can help me.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [ ] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

**Comment by benmccann** [comment]
What do you mean you can't connect to the repro? Do you mean https://kit-svelte-dev-n29rspmw6-svelte.vercel.app/?

What do you mean that you can't sync your commits? What are you trying to do and what error do you get?

What does this PR change? I don't notice anything that looks different visually

**Review by benmccann — APPROVED** [review]

File: sites/kit.svelte.dev/src/lib/search/SearchBox.svelte [hunk 1]
```diff
@@ -238,11 +239,16 @@
...
 	input:focus-visible {
+		--c: white;
 		background: var(--second);
-		color: white;
+		color: var(--c);
 		outline: none;
 	}
 
+	input::placeholder {
+		color: var(--c);
```
Comment (Rich-Harris) [inline sites/kit.svelte.dev/src/lib/search/SearchBox.svelte:249]: I think this should be faded, in the way that placeholders are by default on a white background — something like `rgba(255, 255, 255, 0.5)` (but only when `:focus-visible`, obviously)

**Review by Rich-Harris — CHANGES_REQUESTED** [review]
Thank you! As commented inline, the placeholder shouldn't be the same color as the actual text — it ought to be faded. We could achieve this either with a new selector...

```css
input:focus-visible::placeholder {
  color: rgba(255, 255, 255, 0.5);
}
```

...or an additional custom property, which should be called something descriptive (`--c` is unnecessarily terse — `--color` and `--placeholder` would be better).

**Comment by Rich-Harris** [comment]
Closing in favour of #3981

**Pull request closed without merging**